extensions/amp-consent/0.1/test/test-consent-policy-manager.js

```diff
@@ -576,6 +581,37 @@ describes.realWin(
       });
     });
 
+    describe('getConsentMetadataInfo', () => {
+      let manager;
+
+      beforeEach(() => {
+        manager = new ConsentPolicyManager(ampdoc);
+        manager.setLegacyConsentInstanceId('ABC');
+        env.sandbox
+          .stub(ConsentPolicyInstance.prototype, 'getReadyPromise')
+          .callsFake(() => {
+            return Promise.resolve();
+          });
+        consentInfo = constructConsentInfo(
+          CONSENT_ITEM_STATE.ACCEPTED,
+          'test',
+          {}
+        );
+      });
+
+      it('should return gdprApplies value', async () => {
```

Comment:
nit: Can we add a TODO here, so that we remember to include the gdprApplies value later.

Reply:
sg